src/js/jsx/sections/export/ExportPanel.jsx

```diff
@@ -26,16 +26,33 @@ define(function (require, exports, module) {
     "use strict";
 
     var React = require("react"),
-        classnames = require("classnames");
+        Fluxxor = require("fluxxor"),
+        FluxMixin = Fluxxor.FluxMixin(React),
+        StoreWatchMixin = Fluxxor.StoreWatchMixin,
+        classnames = require("classnames"),
+        _ = require("lodash");
 
     var os = require("adapter/os");
 
     var TitleHeader = require("jsx!js/jsx/shared/TitleHeader"),
         LayerExports = require("jsx!js/jsx/sections/export/LayerExports"),
+        Button = require("jsx!js/jsx/shared/Button"),
+        SVGIcon = require("jsx!js/jsx/shared/SVGIcon"),
         strings = require("i18n!nls/strings"),
-        synchronization = require("js/util/synchronization");
+        synchronization = require("js/util/synchronization"),
+        collection = require("js/util/collection");
+
+    /**
+     * A simple array of scale values, used to determine "next" scale when adding a new asset
+     * @private
+     * @type {Array.<number>}
+     */
+    var _allScales = [0.5, 1, 1.5, 2];
```

Comment:
Maybe in the next iteration you could put the supported scales and formats in the export store so that this, e.g., wouldn't need to be duplicated in two different UI components?